packages/mantine/src/components/table/Table.tsx

```diff
@@ -255,3 +206,4 @@ Table.CollapsibleColumn = TableCollapsibleColumn;
 Table.AccordionColumn = TableAccordionColumn;
 Table.DateRangePicker = TableDateRangePicker;
 Table.Consumer = TableConsumer;
+Table.Loading = TableLoading;
```

Comment:
I think we should also re-export the Layouts here, it would be nicer to import/use Table.Layouts